src/isIsogram.test.js

```diff
@@ -6,4 +6,39 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+  
+  it('should return true for an empty string', () => {
+    const result = isIsogram('');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return true for a lowercase isogram', () => {
+    const result = isIsogram('playgrounds');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return false for a mixed-case word', () => {
+    const result = isIsogram('Adam');
```

Comment:
The test case 'should return false for a mixed-case word' is incorrect because the function is supposed to be case-insensitive. Consider using a test case with a mixed-case isogram to verify the function correctly handles case.